### DISCUSSION TITLE
[ FF8 ] FF VIII "Mega" Mods for FF8 FFNX!

### DISCUSSION THREAD

**Discussion by someyoungguy00** [julianxhokaxhiu/FFNx#698] [Show and tell]
## Summary

I have converted some mod packs by use in the legacy (Tonberry) method please see my guide in Steam community
https://steamcommunity.com/sharedfiles/filedetails/?id=3252361363

### Motivation

Please keep some of the naming and format the same as used by FFNx-v1.19.1.0 for the following catagories
Menu
World
Triple Triad
Battle models both enemy and character
Field models both enemy and character

**Comment by myst6re** [comment]
Hello, for the override per texture, I will keep the names as it. For the override per vram page, currently only available in canary, the names are sill in discussion

**Reply by someyoungguy00** [reply]
Cool that works i will try to update accordingly!
You can shorten them as well without the .dat and .tim extensions *OBVIOUSLY i should be able to pick up on that

Just wanted to show some love seeing the ECHO-S voice mod is out and no one has taken the time to do much more yet, looking forward to the next Stable release.

**Reply by someyoungguy00** [reply]
Ya great to see more support for FF8 textures in the latest Canary

Some suggestions about support for textures (you can follow them or not doesn't matter):

Menu - folder and filename stucture is already pretty standard, should be easy to import from legacy packs
Triple Triad - You may want to break up the files to the legacy method, smaller and easier to import. Folder structure looks good

World - Probably go with the legacy method for the smaller textures like the railway 16x16 where they are all in one .PNG it seems to slow the game down in the world map with custom tetures in the current method - both /world/esk/ folder and /world/dat/wmset/ the other /world/dat/texl/ files currently have no impact (textures for the grass/mountain etc.)

Battle Models and Feild models seem to work just fine no legacy support needed it is probably better to have them in the full .png picture instead of in a 512x512 area of tonberry's 1024x1024 .png which is tonberry's legacy default size.

Also shortening some names might be nice to more closely match the .dat textures in the .fs archives. Without the extension in the name.

**Reply by someyoungguy00** [reply]
I also got in contact with MCindus and my packs will soon be obsolete when the texture support is finished and my guide will be using his packs next. He has said he is going to also have support for FFNx 2013 release with some new texture packs so if you would like to preview how the game might look when finished follow my guide until he finishes his work. Tested with 1.19.1.0

Renamed guide to FF VIII "Ultimate" Mods for FF8 FFNX! to more closely match the 2019 legacy guide made by him which this is a continuation of his as pretty much all Tonberry's legacy mods can now be used in FFNx and i hope this will draw more attention to this great projects. Thanks
